### PR TITLE
[PowerToys]Add docs for Crop And Lock

### PR DESCRIPTION
**Pull request by jaimecbernardo**
Adds documentation for the new Crop And Lock utiltity in PowerToys.

### CODE REVIEW

**Comment by jaimecbernardo** [comment]
To be merged only with the PowerToys 0.73 release if Crop And Lock ships in it.

**Review by alvinashcraft — COMMENTED** [review]
Overall, this looks good to me. In the feature, is the "And" in "Crop And Lock" capitalized? If not, I recommend changing the "and" to be lower-case everywhere in the docs. We're waiting for the next PT release to publish this, correct?

**Comment by jaimecbernardo** [comment]
> Overall, this looks good to me. In the feature, is the "And" in "Crop And Lock" capitalized? If not, I recommend changing the "and" to be lower-case everywhere in the docs. We're waiting for the next PT release to publish this, correct?

Thanks for the review. Indeed we're using "Crop And Lock" currently in the utility. But someone there's a "Crop & Lock" suggestion too. https://github.com/microsoft/PowerToys/pull/27832

Yes, the idea is to wait for the next PT release. Thank you!

File: hub/powertoys/crop-and-lock.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+### Enable
+
+To start using Crop And Lock, enable it in the PowerToys Settings (**Crop And Lock** tab).
+
+### How to use
+
+Once enabled, you can focus a Window and press the "Thumbnail" shortcut (default: <kbd>⊞ Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>T</kbd>) or the "Reparent" shorcut (default: <kbd>⊞ Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>R</kbd>) to select an area of the window to crop.
+
+After you've selected the area of the window, a new window will appear and behave according to the selected crop mode.
```
Comment (Jay-o-Way) [inline hub/powertoys/crop-and-lock.md]: Let's merge these?

Comment (jaimecbernardo) [reply hub/powertoys/crop-and-lock.md]: Just did. Thank you for the suggestion!

**Review by alvinashcraft — APPROVED** [review]

**Pull request merged**